docs/examples/relations.rst

```diff
@@ -1,8 +1,31 @@
 Theory of Relations
 ===================
 
+This simple family example demonstrates theory of finite relations extension to the theory of finite sets.
+Relations are defined as sets of tuples with arity :math:`\geq 1`.
+We have unary relations :math:`people, males,` and :math:`females` of arity 1.
+We also have binary relations :math:`father, mother, parent, ancestor`, and :math:`descendant` of arity 2.
+Here we follow a prefix convention (i.e., :math:`R(x,y)` or
+:math:`\langle x, y \rangle \in R` if and only if
+person :math:`x` has relation `R` with person :math:`y`,
+where :math:`R \in \{father, mother, parent, ancestor, descendant\}`.
+
+
+We have the following list of constraints:
+
+- All relations are nonempty.
+- People is the universe set.
+- Males and females are disjoint sets (i.e., :math:`males \cap females = \phi`).
+- Fathers are males (i.e., :math:`father \bowtie people \subseteq males`).
+- Mothers are females (i.e., :math:`mother \bowtie people \subseteq females`).
+- A parent is a father or a mother (i.e., :math:`parent = father \cup mother`).
+- Ancestor relation is the transitive closure of parent (i.e., :math:`ancestor = parent^{+}`).
+- Descendant relation is the transpose of ancestor (i.e., :math:`descendant = ancestor^{-1}`).
+- No self ancestor (i.e., :math:`\forall x: Person. \langle x, x \rangle \not\in ancestor`).
```

Comment:
I agree both syntax and semantics of these operators need to be defined in the theory page, but 
the mathematical formulation here is related to the example, and I think it is important to show the mathematical notation. 
Without them, one would express the constraint `fathers are males` using quantifiers instead of the appropriate relational join operator.

Reply:
I added a footnote for the bowtie symbol with the citation paper.

Reply:
I prefer to address the definitions of relational operators in a separate PR.

Reply:
Okay, that's fine.